packages/@atjson/source-html/src/converter/social-embeds.ts

```diff
@@ -21,7 +21,8 @@ function isBlockquoteEmbed(annotation: Annotation<any>) {
   return (
     classList.includes("instagram-media") ||
     classList.includes("twitter-tweet") ||
-    classList.includes("tiktok-embed")
+    classList.includes("tiktok-embed") ||
+    classList.includes("threads-embed")
```

Comment:
Can we add a test for the Threads embed? Besides checking that the conversion is correct, this will document what we expect the Threads html embed code to look like

Reply:
done